packages/react-dom/src/events/replayedEvent.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
This file basically only exists to avoid a circular dependency between DOMEventPlugin and ReactDOMEventReplaying